docs/story.rst.j2

```diff
@@ -0,0 +1,153 @@
+{#
+    A Jinja2 template for rendering stories in tmt's own docs.
+
+    The template is based on a template bundled with tmt as an
+    example on template/rst export. The difference is how links
+    are rendered: the default template renders just the very basic
+    info about idea vs. implemented/verified, this template tries
+    to render links related to tmt docs and sources as usable HTTP
+    links.
+#}
+
+{% macro render_example(example) %}
+    {% set example = example.strip() %}
+    {% set syntax = example | regex_search('^# syntax: ([a-z]+)') %}
+    {% if syntax %}
+        {% set example = '\n'.join(example.splitlines()[1:]) %}
+        {% set syntax = syntax[0] %}
+    {% else %}
+        {% set syntax = 'yaml' %}
+    {% endif %}
+
+.. code-block:: {{ syntax }}
+
+   {{ example | indent(width=3, first=False, blank=False) | trim }}
+{% endmacro %}
+
+{#
+    Convert link relation into a nicely formatted string.
+
+    :param link: tmt.base.Link instance.
+#}
+{% macro printable_relation(link) %}{{ link.relation.replace('relates', 'relates-to').replace('-', ' ').capitalize() }}{% endmacro %}
+
+{#
+    Emit a link leading to tmt's upstream repository.
+
+    :param link: tmt.base.Link instance.
+    :param pattern: used to extract link label from its target. It shall include
+        one capturing group, representing the label.
+#}
+{% macro emit_tmt_link(link, pattern) %}
+* {{ printable_relation(link) }} `{{ link.target | regex_replace(pattern, '\\1') }} <https://github.com/teemtee/tmt/tree/main/{{ link.target }}>`_ {% if link.note %}({{ link.note }}){% endif %}
```

Comment:
Generated links contain a double-slash `//` in them, for example:

    https://github.com/teemtee/tmt/tree/main//tmt/steps/provision/podman.py

Could we strip the extra slash?

Reply:
Fixed in 2c343d76

Reply:
Thanks!